### PR TITLE
ISSD-1329 Search Result page to be faceted by Capabilities, Features & Deployment Method

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,334 @@
+<#macro treeview_item
+	cssClassTreeItem = ""
+	frequency = 0
+	id = ""
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:4]: sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+			}
+		}
+
+		document.getElementById(id).classList.add('d-none');
+
+		sessionStorage.setItem(id, true);
+	}
+</@>
+
+<style>
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:268]: move the style to the start of the article

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by abfranca** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+<#macro treeview_item
+	cssClassTreeItem = ""
+	frequency = 0
+	frequencyVisible = true
+	id = ""
+	name = ""
+	selectable = false
+	selected = false
+	termDisplayContexts = ""
+	termDisplayContextClass = ""
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:78]: sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+						<#break>
+					</#if>
+				</#list>
+			</div>
+
+			<div class="collapse show" id="${namespace}treeItem${id}">
+				<ul class="treeview-group" role="group">
+					<#assign
+						termDisplayContextCount = 1
+						hasTermDisplayContextHidden = false
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:195]: sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+					/>
+
+					<#list termDisplayContexts as termDisplayContext>
+						<#assign hideClass = "" />
+
+						<#if termDisplayContextCount gt 8 && !termDisplayContext.isSelected()>
+							<#assign
+								hideClass = "${vocabularyName}-class d-none"
+								hasTermDisplayContextHidden = true
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:204]: sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl [hunk 6]
```diff
@@ -1,8 +1,138 @@
...
+	}
+
+	.search-results .search-results-entry .search-results-entry-title .modified-date {
+		color: var(--neutral-07, #6C6C75);
+		font-family: 'Source Sans Pro', sans-serif;
+		font-size: 0.8125rem;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 1rem;
+	}
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl:128]: sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl [hunk 7]
```diff
@@ -12,15 +142,31 @@
...
-					</a>
+						<div class="description search-results-entry-content">
+							${searchEntryContent}
+						</div>
+
+						<div class="modified-date pt-2">
+							<#assign
+								modifiedDateStringParts = entry.getModifiedDateString()?split(":")
+								finalModifiedDateString = ""
+							/>
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl:155]: sort

**Review by nNilton — CHANGES_REQUESTED** [review]

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl (hunk 3 above)
Comment (abfranca) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:77]: Done.

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl (hunk 5 above)
Comment (abfranca) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-category-facet/ddm-template.ftl:203]: Done.

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl (hunk 6 above)
Comment (abfranca) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl:128]: Done.

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl (hunk 7 above)
Comment (abfranca) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-results/ddm-template.ftl:155]: Done.

**Comment by abfranca** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/145213

**Pull request closed without merging**